File: templates/_composition/MVVMBasic/Feature.SuspendAndResume/Services/SuspendAndResumeService_postaction.cs
using System.Reflection;
using Windows.UI.Xaml.Controls;
namespace ItemNamespace.Services
{
    internal class SuspendAndResumeService : ActivationHandler<LaunchActivatedEventArgs>
    {
        private async Task RestoreStateAsync()
        {
            //^^
            if (typeof(Page).IsAssignableFrom(saveState?.Target))
            {
                NavigationService.Navigate(saveState.Target, saveState.SuspensionState);
            }
        }
    }
}

File: templates/Uwp/_composition/_shared/Feature.SuspendAndResume.AddNavigationVB/Services/SuspendAndResumeService_postaction.vb
'{[{
Imports System.Reflection
'}]}
Namespace Services
    Friend Class SuspendAndResumeService
        Inherits ActivationHandler(Of LaunchActivatedEventArgs)

        Private Async Function RestoreStateAsync() As Task
            '^^
            '{[{
            If saveState?.Target IsNot Nothing AndAlso GetType(Page).IsAssignableFrom(saveState.Target) Then
                NavigationService.Navigate(saveState.Target, saveState.SuspensionState)
            End If
            '}]}
        End Function
    End Class
End Namespace
